tests/test_core.py

```diff
@@ -5522,7 +5522,7 @@ def test_freetype(self):
     assert 'asm2g' in core_test_modes
     if self.run_name == 'asm2g':
       # flip for some more coverage here
-      self.set_setting('ALIASING_FUNCTION_POINTERS', 1 - self.get_setting('ALIASING_FUNCTION_POINTERS'))
```

Comment:
Nit: Should we explicitly set this to 0 for the prior test, to ensure coverage if we start defaulting to 1?

Reply:
It defaults to zero.. which is why i think this change is nop.   I could just revert this line?

Reply:
reverted